lib/src/core/mod.rs

```diff
@@ -66,17 +65,56 @@ impl Attestation {
         let evidence =
             serde_json::from_str::<Evidence>(evidence).context("Deserialize Evidence failed.")?;
         let verifier = TEE::from_str(&evidence.tee)?.to_verifier()?;
-        let results = verifier.evaluate(&evidence, policy, reference_data).await?;
-        let results = serde_json::to_string(&results)?;
-        debug!("Attestation Results: {}", &results);
-        Ok(results)
-    }
 
-    pub fn policy(&self, tee: TEE) -> Result<String> {
-        tee.to_verifier()?.default_policy()
+        let tee_evidence_parsed_claims = match verifier.evaluate(&evidence).await {
```

Comment:
might be more clear as `claims_parsed_from_tee_evidence` or just `claims_from_evidence`. Maybe also change the type alias above.